qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/customers/CustomerDbConnector.groovy

```diff
@@ -124,4 +137,276 @@ class CustomerDbConnector implements CreateCustomerDataSource, UpdateCustomerDat
       throw new DatabaseQueryException("The affiliation could not be created: ${affiliation.toString()}")
     }
   }
+
+  /*TODO everything below was copied over from DatabaseQueries*//*
+
+  *//**
```

Comment:
I would not do that. I fear these classes will be forgotten when we leave them like this. I would either delete them or just leave them there. Afaik the implemented methods we had worked :)

Reply:
I pasted it in here to stepwise migrate the methods so that the functionality is the same without having to have a Querries class. This PR is not finished but I wanted your feedback before I went ahead.